chain/messagepool/messagepool.go

```diff
@@ -45,7 +45,7 @@ var (
 )
 
 const (
-	msgTopic = "/fil/messages"
+	msgTopic = "/fil/msgs"
```

Comment:
From [spec](https://filecoin-project.github.io/specs/#systems__filecoin_nodes__network) this should include the network name on the msgTopic: `fil/msgs/<network-name>`